services/provider/proto/provider.proto

```diff
@@ -73,6 +73,8 @@ message ExternalResource {
     string kind = 2;
     // Data contains the contents of the external cluster resource
     bytes Data = 3;
+    // ApiVersion contains Group and Version of the resource
+    string apiVersion = 4;
```

Comment:
If you are introducing this field in this PR you should add the correct value for all payloads who are sending `ExternalResource`

Reply:
Removed from this PR and I'll introduce that as a separate one.